maps/away/scenarios/enviro_testing_facility/enviro_testing_facility_markers.dm

```diff
@@ -0,0 +1,115 @@
+
+// ---
+
+/obj/effect/map_effect/marker/mapmanip/submap/extract/enviro_testing_facility/main/crew_quarters
+	name = "main/crew_quarters"
+
+/obj/effect/map_effect/marker/mapmanip/submap/insert/enviro_testing_facility/main/crew_quarters
+	name = "main/crew_quarters"
+
+// ---
+
+/obj/effect/map_effect/marker/mapmanip/submap/extract/enviro_testing_facility/main/offices
+	name = "main/offices"
+
+/obj/effect/map_effect/marker/mapmanip/submap/insert/enviro_testing_facility/main/offices
+	name = "main/offices"
+
+// ---
+
+/obj/effect/map_effect/marker/mapmanip/submap/extract/enviro_testing_facility/main/canteen
+	name = "main/canteen"
+
+/obj/effect/map_effect/marker/mapmanip/submap/insert/enviro_testing_facility/main/canteen
+	name = "main/canteen"
+
+// ---
+
+/obj/effect/map_effect/marker/mapmanip/submap/extract/enviro_testing_facility/main/dome
+	name = "main/dome"
+
+/obj/effect/map_effect/marker/mapmanip/submap/insert/enviro_testing_facility/main/dome
+	name = "main/dome"
+
+// ------------------------------ misc submaps
+
+/obj/effect/map_effect/marker/mapmanip/submap/extract/enviro_testing_facility/misc/armory
+	name = "cargo/armory"
+
+/obj/effect/map_effect/marker/mapmanip/submap/insert/enviro_testing_facility/misc/armory
+	name = "cargo/armory"
+
+// ---
+
+/obj/effect/map_effect/marker/mapmanip/submap/extract/enviro_testing_facility/misc/engineering_storage
+	name = "cargo/engineering_storage"
+
+/obj/effect/map_effect/marker/mapmanip/submap/insert/enviro_testing_facility/misc/engineering_storage
+	name = "cargo/engineering_storage"
+
+// ---
+
+/obj/effect/map_effect/marker/mapmanip/submap/extract/enviro_testing_facility/misc/science_storage
+	name = "cargo/science_storage"
+
+/obj/effect/map_effect/marker/mapmanip/submap/insert/enviro_testing_facility/misc/science_storage
+	name = "cargo/science_storage"
+
+// ---
+
+/obj/effect/map_effect/marker/mapmanip/submap/extract/enviro_testing_facility/misc/crew_room_small
+	name = "cargo/crew_room_small"
+
+/obj/effect/map_effect/marker/mapmanip/submap/insert/enviro_testing_facility/misc/crew_room_small
+	name = "cargo/crew_room_small"
+
+// ---
+
+/obj/effect/map_effect/marker/mapmanip/submap/extract/enviro_testing_facility/misc/crew_room_big
+	name = "cargo/crew_room_big"
+
+/obj/effect/map_effect/marker/mapmanip/submap/insert/enviro_testing_facility/misc/crew_room_big
+	name = "cargo/crew_room_big"
+
+// ---
+
+/obj/effect/map_effect/marker/mapmanip/submap/extract/enviro_testing_facility/misc/crew_lounge
+	name = "cargo/crew_lounge"
+
+/obj/effect/map_effect/marker/mapmanip/submap/insert/enviro_testing_facility/misc/crew_lounge
+	name = "cargo/crew_lounge"
+
+// ---
+
+/obj/effect/map_effect/marker/mapmanip/submap/extract/enviro_testing_facility/misc/warehouse_room
+	name = "cargo/warehouse_room"
+
+/obj/effect/map_effect/marker/mapmanip/submap/insert/enviro_testing_facility/misc/warehouse_room
+	name = "cargo/warehouse_room"
+
+// ------------------------------ fin
+
```

Comment:
https://github.com/Aurorastation/Aurora.3/blob/master/.github/guides/STYLE.md#use-comments-where-possible-and-sensible

`Do not use comments to mark obvious things, like the start or end of a file, unless they document the content of the file.`